src/wallet/db.cpp

```diff
@@ -408,8 +408,8 @@ bool BerkeleyBatch::VerifyEnvironment(const fs::path& file_path, std::string& er
     LogPrintf("Using wallet %s\n", file_path.string());
 
     // Wallet file must be a plain filename without a directory
-    if (walletFile != fs::basename(walletFile) + fs::extension(walletFile))
-    {
+    fs::path wallet_file_path = fs::path(walletFile);
```

Comment:
nit: could construct directly

Reply:
@Empact Thank you for your review.
I'm not sure if I understand your comment correctly. Could you elaborate, please?

Reply:
```diff
-fs::path wallet_file_path = fs::path(walletFile);
+fs::path wallet_file_path(walletFile);
```

Reply:
Thank you! Fixed.